.gitignore

```
@@ -1,2 +1,5 @@
 Candidate/appsettings.json 
 Candidate/appsettings.local.json
+bin
+obj
+.vs
```

Note, you'll also need to now delete all the files in these directories to clean up the repo. 
```suggestion
.vs/
```